esbuild.js

```diff
@@ -0,0 +1,13 @@
+const { getOptions } = require('./scripts/build/utils/options.js');
```

Comment:
sure I think it could be helpful for you two to start and then we can go from there

and yeah, this is the same `getOptions`, I figured we already have a good data structure there for this so no need to reinvent the wheel